certificate/forms.py

```diff
@@ -16,3 +17,23 @@ class FeedBackForm(forms.Form):
     #            (max_length=30, required=False)
     #state = forms.CharField\
     #            (max_length=128)
+
+email_subject_choice = [
+('Certificate not Awarded','Certificate not Awarded'),
+('Change in Name','Change in Name'),
+('Issue with Workshop Date','Issue with Workshop Date'),
+('Invalid Email Address','Invalid Email Address'),
+('Others','Others')]
+ws_type_choice = [
+('iscp','Introduction to Scientific Computing using Python(ISCP)'),
+('3day','Basic Programming using Python'),
+('sel','Self Learning(Basics of Python)')]
+
+class ContactForm(forms.Form):
+    name = forms.CharField(label= 'Full Name',max_length=30)
+    email = forms.EmailField()
+    date = forms.DateField(label="Workshop Date",initial=datetime.date.today)
+    category = forms.CharField(widget=forms.Select(choices=ws_type_choice))
```

Comment:
Is this editable dropdown?
I mean user can write his own choice in category or subject!

Reply:
No , there fields are fixed drop-downs. In email subject they can select "Other" if required to address a different issue